root/usr/share/nethesis/NethServer/Module/Account/Group/Modify.php

```diff
@@ -158,6 +161,29 @@ public function process()
         }
 
         $this->getPlatform()->signalEvent(sprintf('group-%s', $event), $params);
+
+        // Email group alias creation
+        if (($this->getIdentifier() === 'create') && ($this->parameters['CreatePseudoRecords'] === 'yes')) {
+
+            $accountsDb = $this->getPlatform()->getDatabase('accounts');
+            $domain = $this->getPlatform()->getDatabase('configuration')->getType('DomainName');
+            $groupnameFull = $this->parameters['groupname'] . '@' . $domain;
+            $members= array_unique($this->parameters['members']);
+            $members[] = $groupnameFull;
+            $members = implode(",",$members);
+
+            if( ! $accountsDb->getKey($groupnameFull)) {
+                $accountsDb->setKey($groupnameFull, 'pseudonym', array ('Description'=>'Automatic group mailbox', 'Account' => $members));
```

Comment:
TL;DR remove `$members` variable, set "Account" to `vmail+$this->parameters['groupname']`

----

Well, I checked again our code [here](https://github.com/NethServer/nethserver-mail/blob/master/server/usr/share/nethesis/NethServer/Module/MailAccount/Pseudonym/Modify.php#L173) and I realized I put you in the wrong direction.

There is just one destination/target "Account"  for our pseudonym. When the destination is a shared mailbox, its syntax uses an extension address, like `vmail+mbxname`, without domain part.

To clarify, this is an example:

```text
repomaster@domain.tld=pseudonym
    Access=public
    Account=vmail+repomaster
    ControlledBy=system
    Description=
```